tests/tips/test_feed_suggestion.py

```diff
@@ -70,6 +78,9 @@ async def test_fake_queryid_feed_setup(autopay_contract_setup, caplog):
         _amount=int(1 * 10**18),
     )
     assert status.ok
-    datafeed, tip = await get_feed_and_tip(autopay)
+    datafeed, tip = await get_feed_and_tip(flex.autopay)
     assert datafeed is None
     assert tip is None
+    assert (
+        "No feeds to report, all funded feeds w/threshold gt zero and no API support to check if threshold is met"
```

Comment:
What's this mean?
`all funded feeds w/threshold gt zero and no API support to check if threshold is met`
Hard to understand for me